dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/scalaUtils/PythonUnion.scala

```diff
@@ -5,14 +5,38 @@ import scala.language.implicitConversions
 import scala.reflect.ClassTag
 
 object PythonUnion {
-  def convert[X <: py.Any, Y <: py.Any, A <: PythonType[X], B <: PythonType[Y]](u: A | B)(implicit ev1: ClassTag[A], ev2: ClassTag[B]): py.|[X, Y] =
+
+  def fromPyhtonTypeAndScalaTypeUnion[X <: py.Any, A <: PythonType[X], B <: Any](u: A | B)(implicit ev1: ClassTag[A], ev2: ClassTag[B]): py.|[B, X] =
```

Comment:
This cannot be implicit conversion without some clever type bound on `B` because it collides with the one beneath `fromPyhtonTypesUnion `